Model/ProductExport.php

```diff
@@ -392,21 +426,22 @@ protected function addValueToDataArray(&$data, $key, $value)
 
     protected function setProductPrices($product, &$data, &$childProducts = null)
     {
-        $basePrices = $this->getProductPrice($product, false, true, $childProducts);
-        $baseFinalPrices = $this->getProductPrice($product, true, true, $childProducts);
+        // $basePrices = $this->getProductPrice($product, false, true, $childProducts);
+        // $baseFinalPrices = $this->getProductPrice($product, true, true, $childProducts);
+        $prices = $this->getProductPrices($product, true, $childProducts);
```

Comment:
Optimizations to prevent it double looping around products to get base prices and base final prices